06_puppy/n0npax/types.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"sync"
+)
+
```

Comment:
Great work separating the types out into their own file. Are you able to add some documentation to these exported types? It will be good to get in the habit of documenting any exported types, interfaces and functions as we start to move towards building an API.

General convention is a comment above the exported type/function, which starts with the type/function name. Eg.

```go
// Puppy contains information about a single Puppy ...
type Puppy struct {
    ...
}
```

Reply:
Fixed. Thanks!